app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -401,6 +413,9 @@ public void onRlContainerTitleClicked() {
      * @param shouldExpand
      */
     private void expandCollapseLlMediaDetail(boolean shouldExpand){
+        if(isExpanded==shouldExpand){
```

Comment:
needs more space to adhere to codestyle, also can we rename this method? the `Ll` I am guessing stands for LinearLayout but it just looks so fugly

Reply:
Hmm, can you suggest a better name?

Reply:
just getting rid of the `Ll` is fine I would think

Reply:
@macgills  Done :)